src/Codeception/Module/Asserts.php

```diff
@@ -263,35 +262,6 @@ public function assertFileNotExists($filename, $message = '')
     }
 
     /**
-     * @param $expected
```

Comment:
yep, you are right. Scrutinzer has suggested that we had those methods written in wrong way